bofire/strategies/predictives/botorch.py

```diff
@@ -214,6 +217,13 @@ def has_sufficient_experiments(
         return False
 
     def get_acqf_input_tensors(self):
+        """
+
+        Returns:
+            X_train (Tensor): Tensor of shape (n, d) with n training points and d input dimensions.
```

Comment:
Thanks for adding the docstring. I would prefer to have the type information as type annotation instead of in the docstring. I.e.,
```python
def get_acqf_input_tensors(self) -> tuple[Tensor, Tensor | None]:
    """
    Returns:
            X_train: Tensor of shape (n, d) with n training points and d input dimensions
            X_pending: Tensor of shape (m, d) with m pending points
    """
```